packages/@lwc/engine-core/src/renderer.ts

```diff
@@ -170,114 +116,101 @@ type setCSSStylePropertyFunc = (
-    getChildren = getChildrenImpl;
-}
 
 type getChildNodesFunc = (element: E) => NodeList;
 export let getChildNodes: getChildNodesFunc;
-export function setGetChildNodes(getChildNodesImpl: getChildNodesFunc) {
-    getChildNodes = getChildNodesImpl;
-}
 
 type getFirstChildFunc = (element: E) => N | null;
 export let getFirstChild: getFirstChildFunc;
-export function setGetFirstChild(getFirstChildImpl: getFirstChildFunc) {
-    getFirstChild = getFirstChildImpl;
-}
 
 type getFirstElementChildFunc = (element: E) => E | null;
 export let getFirstElementChild: getFirstElementChildFunc;
-export function setGetFirstElementChild(getFirstElementChildImpl: getFirstElementChildFunc) {
-    getFirstElementChild = getFirstElementChildImpl;
-}
 
 type getLastChildFunc = (element: E) => N | null;
 export let getLastChild: getLastChildFunc;
-export function setGetLastChild(getLastChildImpl: getLastChildFunc) {
-    getLastChild = getLastChildImpl;
-}
 
 type getLastElementChildFunc = (element: E) => E | null;
 export let getLastElementChild: getLastElementChildFunc;
-export function setGetLastElementChild(getLastElementChildImpl: getLastElementChildFunc) {
-    getLastElementChild = getLastElementChildImpl;
-}
 
 type isConnectedFunc = (node: N) => boolean;
 export let isConnected: isConnectedFunc;
-export function setIsConnected(isConnectedImpl: isConnectedFunc) {
-    isConnected = isConnectedImpl;
-}
 
 type insertGlobalStylesheetFunc = (content: string) => void;
 export let insertGlobalStylesheet: insertGlobalStylesheetFunc;
-export function setInsertGlobalStylesheet(insertGlobalStylesheetImpl: insertGlobalStylesheetFunc) {
-    insertGlobalStylesheet = insertGlobalStylesheetImpl;
-}
 
 type insertStylesheetFunc = (content: string, target: ShadowRoot) => void;
 export let insertStylesheet: insertStylesheetFunc;
-export function setInsertStylesheet(insertStylesheetImpl: insertStylesheetFunc) {
-    insertStylesheet = insertStylesheetImpl;
-}
 
 type assertInstanceOfHTMLElementFunc = (elm: any, msg: string) => void;
 export let assertInstanceOfHTMLElement: assertInstanceOfHTMLElementFunc;
-export function setAssertInstanceOfHTMLElement(
-    assertInstanceOfHTMLElementImpl: assertInstanceOfHTMLElementFunc
-) {
-    assertInstanceOfHTMLElement = assertInstanceOfHTMLElementImpl;
-}
 
 type defineCustomElementFunc = (
     name: string,
     constructor: CustomElementConstructor,
     options?: ElementDefinitionOptions
 ) => void;
 export let defineCustomElement: defineCustomElementFunc;
-export function setDefineCustomElement(defineCustomElementImpl: defineCustomElementFunc) {
-    defineCustomElement = defineCustomElementImpl;
-}
 
 type getCustomElementFunc = (name: string) => CustomElementConstructor | undefined;
 export let getCustomElement: getCustomElementFunc;
-export function setGetCustomElement(getCustomElementImpl: getCustomElementFunc) {
-    getCustomElement = getCustomElementImpl;
+
+export interface RendererAPI {
```

Comment:
this is just the API that was previously defined in renderer.ts shape, but since it needs to be passed around, I Don't know if there is a way to define a type based on the shape of a module that can be imported. In any case, I did it manually.